src/engine/init.c

```diff
@@ -425,6 +424,8 @@ set_abt_max_num_xstreams(int n)
 	return 0;
 }
 
+static bool d_abt_on;
```

Comment:
Why did you introduce this ?? Did you trigger some issue (deadlock ?) due to early logging ??

Reply:
Arbobots hits this assert if compiled with debugging without this change.
https://github.com/pmodels/argobots/blob/main/src/self.c#L135